app/assets/stylesheets/bali/forms.scss

```
@@ -93,3 +93,48 @@
     margin-bottom: 0.5rem;
   }
 }
+
+.radio-buttons-grouped {
+  $togglers-margin-bottom: 1.5rem;
+
+  $toggler-margin-right: 1rem;
+  $toggler-border: solid 1px;
```

creo que falta el color del border

```suggestion
  $toggler-border: 1px solid color;
```